packages/xo-web/src/xo-app/backup/index.js

```diff
@@ -23,22 +23,26 @@ import Page from '../page'
 
 const HealthNavTab = decorate([
   addSubscriptions({
-    // used by createGetLoneSnapshots
+    // used by getLoneSnapshots
     schedules: subscribeSchedules,
+    jobs: subscribeBackupNgJobs,
   }),
   connectStore({
-    nLoneSnapshots: createGetLoneSnapshots.count(),
+    loneSnapshots: getLoneSnapshots,
```

Comment:
No need to change the current implementation because we only display the number of lone snapshots. So, you can use `selectors/createCounter`.